python/vitables/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                vitables
+version             3.0.3
+categories          python science
+platforms           darwin
```

Comment:
please change to 
`platforms           {darwin any}`
and add 
`supported_archs     noarch`

since the package doesn't compile anything and/or installs architecture-dependent files